### PR TITLE
K8SPG-588: Add trackLatestRestorableTime

### PR DESCRIPTION
**Pull request by egegunes**

### CODE REVIEW

**Comment by hors** [comment]
@egegunes I think we can't use main :( we need to have some release branch  @jvpasinatto WDYT?

**Comment by jvpasinatto** [comment]
@egegunes @hors Yes, please commit in [K8SPG-604-pg-250](https://github.com/percona/percona-helm-charts/tree/K8SPG-604-pg-250) branch, so we can try to bump the chart version only once to 2.5.0

File: charts/pg-db/values.yaml [hunk 1]
```diff
@@ -311,6 +311,7 @@ proxy:
 #          pool_mode: transaction
 
 backups:
+  trackLatestRestorableTime: true
```
Comment (jvpasinatto) [inline charts/pg-db/values.yaml:314]: Please add this parameter in the readme

**Review by jvpasinatto — CHANGES_REQUESTED** [review]

File: charts/pg-db/values.yaml (hunk 1 above)
Comment (egegunes) [reply charts/pg-db/values.yaml:314]: https://github.com/percona/percona-helm-charts/pull/366/commits/7dd7151278231e4499304e488f95e51f3e0b1cad

**Review by jvpasinatto — APPROVED** [review]

**Pull request merged**